mne/beamformer/_lcmv.py

```diff
@@ -482,10 +540,29 @@ def _prepare_beamformer_input(info, forward, label, picks, pick_ori):
     if info['projs']:
         G = np.dot(proj, G)
 
-    # Pick after applying the projections
+    # Pick after applying the projections. This makes a copy of G, so further
+    # operations can be safely done in-place.
     G = G[picks_forward]
     proj = proj[np.ix_(picks_forward, picks_forward)]
 
+    # Normalize the leadfield if requested
+    if fwd_norm == 'dipole':
+        G /= np.linalg.norm(G, axis=0)
```

Comment:
Is the norm here computed voxel by voxel?

Reply:
that is correct.

Reply:
yeah, makes sense. I misread, my bad!